src/arrayMethodSort.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
+        const current = this[j];
+        const next = this[j + 1];
```

Comment:
```suggestion
    for (let i = 0; i < this.length - 1; i++) {
      for (let j = i; j < this.length; j++) {
        const current = this[i];
        const next = this[j];
```
Use both loops

Reply:
Why? It's just a simple bubble sort
```suggestion
    for (let i = 0; i < this.length; i++) {
      for (let j = 0; j < this.length - i - 1; j++) {
        const current = this[j];
        const next = this[j + 1];
```